geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractInfoIntegrationTest.java

```diff
@@ -182,10 +295,9 @@ public void shouldNotReturnKeySpaceSection_givenServerWithNoKeys() {
   }
 
   @Test
-  public void shouldThrowExceptionIfGivenMoreThanOneParameter() {
+  public void shouldThrowException_ifGivenMoreThanOneParameter() {
     assertThatThrownBy(
         () -> jedis.sendCommand(
             Protocol.Command.INFO, "Server", "Cluster")).hasMessageContaining("ERR syntax error");
```

Comment:
If you end up making changes anyway, we could also change this to `hasMessage()` instead of `hasMessageContaining()` so we check the exact message.